### PR TITLE
lemurs: init at 0.3.1

### LINKED ISSUE #246035: Package request: lemurs
**Project description**

A customizable TUI display/login manager written in Rust. Lemurs provides a Terminal User Interface (TUI) for a Display/Login Managers in Rust for most GNU/Linux and BSD distributions. It can work both with or without SystemD. Lemurs works on most Unix systems including Linux, FreeBSD and NetBSD.

**Metadata**

* homepage URL: https://github.com/coastalwhite/lemurs/
* source URL: https://github.com/coastalwhite/lemurs/
* license: Apache-2.0, MIT
* platforms: unix, linux

### PR DESCRIPTION
**Pull request by JeremiahSecrist**
## Description of changes
Adds lemurs, a tty based display manager. I have tested it runs and accepts inputs.
This resolves #246035 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7724,6 +7724,13 @@
     githubId = 1608697;
     name = "Jens Binkert";
   };
+  jeremiahs = {
```
Comment (drupol) [inline maintainers/maintainer-list.nix:7727]: Keep attributes sorted alphabetically.

File: pkgs/applications/display-managers/lemurs/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  linux-pam,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lemurs";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "coastalwhite";
+    repo = pname;
```
Comment (drupol) [inline pkgs/applications/display-managers/lemurs/default.nix:13]: `repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

**Review by drupol — CHANGES_REQUESTED** [review]
Hi !

Thanks for your contribution.

I've left some comments, left me know if this is clear enough.

**Review by drupol — CHANGES_REQUESTED** [review]
Excellent, thanks !

Please squash your commits so that this PR contains only 2 commits.

- 1 commit for adding you as maintainer
- 1 commit for the rest

**Comment by JeremiahSecrist** [comment]
This should fix all aforementioned issues. Thanks for being patient!

File: pkgs/applications/display-managers/lemurs/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-6mNSLEWafw8yDGnemOhEiK8FTrBC+6+PuhlbOXTGmN0=";
```
[the remark this answers is not in the document]
Comment (drupol) [reply pkgs/applications/display-managers/lemurs/default.nix:15]: This is not fixed.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/applications/display-managers/lemurs/default.nix (hunk 3 above)
[the remark this answers is not in the document]
Comment (JeremiahSecrist) [reply pkgs/applications/display-managers/lemurs/default.nix:15]: Must have been an error on my part with the rebase. Fixing now!

**Review by drupol — APPROVED** [review]
LGTM now

**Pull request merged**